history/taskana-simplehistory-provider/src/test/java/acceptance/events/task/CreateHistoryEventOnTaskTransferAccTest.java

```diff
@@ -35,23 +38,32 @@ class CreateHistoryEventOnTaskTransferAccTest extends AbstractAccTest {
   @WithAccessId(user = "admin")
   @TestFactory
   Stream<DynamicTest> should_CreateTransferredHistoryEvent_When_TaskIsTransferred() {
-    List<Triplet<String, String, Consumer<String>>> testCases =
+    List<Quadruple<String, String, String, Consumer<String>>> testCases =
```

Comment:
Why did you parameterize the old workbasket value? 
What would happen if you'd change the Workbasket of TKI:000000000000000000000000000000000053 to WBI:100000000000000000000000000000000001?

I'd rather have some number change in a test than this added complexity by adding this parameter.

Reply:
as disscussed a comment why the Workbaskets are parameterized is enough.